res/keyboard/en_US.kbd.cfg

```diff
@@ -58,12 +64,18 @@ LoadSelectedTrack Shift+Left
 eject Alt+Shift+Left
 
 loop_in 2
+loop_in_goto Shift+@
 loop_out 3
-reloop_exit 4
+loop_out_goto Shift+#
+reloop_toggle 4
+reloop_andstop Shift+$
 
-beatloop_4_toggle q
+beatloop_activate q
+beatlooproll_activate Shift+q
 loop_halve w
+beatjump_forward Shift+w
```

Comment:
Oops, ``beatjump_forward`` and ``beatjump_backward`` are assigned twice.

I would like to  use the vacant ``Shift+w`` and ``Shift+i`` shortcuts to scale the ``beatjump_size`` inside the spinbox. Does this even work with the keyboard? ``beatjump_size_up`` and ``beatjump_size_down``do not work...

It works with ``loop_halve`` and ``loop_double`` for the `` beatloop_size``, and it would be nice to have something similar for ´´beatjump_size´´